ejb3/pom.xml

```diff
@@ -307,11 +384,25 @@ projects that depend on this project.-->
             <artifactId>hibernate-validator-cdi</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
+        </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>wildfly-weld-common-legacy-namespace</artifactId>
+            <artifactId>wildfly-weld-common</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-commons-jakarta</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-core-jakarta</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
@chengfang I don't think these test dependencies are needed any longer.  Can you verify?

Reply:
@pferraro You're right. I'll let the current test run to complete and remove the 2 deps.